pcl_ros/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(pcl_ros)
 find_package(cmake_modules REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system filesystem thread)
 find_package(Eigen3 REQUIRED)
-find_package(PCL REQUIRED COMPONENTS core features filters io segmentation surface)
+find_package(PCL 1.7 REQUIRED)
```

Comment:
Why you specify 1.7?

Reply:
1. kinetic supports pcl 1.7 http://www.ros.org/reps/rep-0003.html#kinetic-kame-may-2016-may-2021
2. origonall pcl_ros 1.4.1 uses pcl 1.7 https://github.com/ros-perception/perception_pcl/blob/1.4.1/pcl_ros/CMakeLists.txt#L8

Reply:
Ok, so `find_package(PCL REQUIRED)` is fine, right?
I don't think specifying version is not a good idea, because it fobids use of PCL 1.8.
and FYI, in pcl 1.4.0, there was no specification of the version number.

Reply:
it should be ok, but I'm not sure. Can you create new  PR with  `find_package(PCL REQUIRED)` ?

Reply:
Yes